datasetComparison/src/test/scala/za/co/absa/hermes/datasetComparison/ComparisonResultSuite.scala

```diff
@@ -0,0 +1,35 @@
+package za.co.absa.hermes.datasetComparison
+
+import org.apache.spark.sql.Column
+import org.scalatest.FunSuite
+
+class ComparisonResultSuite extends FunSuite {
+
+  val CR1 = ComparisonResult(10, 11, 12, List.empty[Column], None, 13, "--alfa beta")
```

Comment:
Why `CR1` and `CR2` are outside the tests that check it?

Reply:
Had bigger plans, settled for less.